astrokat/test/test_offline_observe.py

```diff
@@ -35,13 +35,7 @@ def test_targets_sim(self):
 
         # get result and make sure everything ran properly
         result = LoggedTelescope.user_logger_stream.getvalue()
-        self.assertIn("target0_radec at azel (179.9, 30.6) deg", result)
-        self.assertIn("target1_radec at azel (179.9, 30.6) deg", result)
-        self.assertIn("target2_gal at azel (345.4, 68.6) deg", result)
-        self.assertIn("target3_azel at azel (10.0, 50.0) deg", result)
-        self.assertIn("target4_azel, tags=radec target, 16:00:05.19 8:50:57.6", result)
-        self.assertIn("target4_azel at azel (10.0, 50.0) deg", result)
-        self.assertIn("Moon at azel (63.4, 66.7) deg", result)
+        self.assertIn("target4_azel, tags=radec target", result)
```

Comment:
@bngcebetsha Removing these checks make the test more lenient.  If we can't have these specific checks when using both `katcorelib` and `astrokat.simulate`, then how about checking it in at least one of the environments?  I.e., only do these checks if we detect `astrokat.simulate` is being used.

We did something like here (although that case was to handle slightly different messages in the two environments)
https://github.com/ska-sa/astrokat/blob/35b20abbfec4f0cbccf1ec92982ae5a289bac57f/astrokat/test/test_scans.py#L102-L112